src/components/select/select.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import type { Meta } from "@storybook/react"
+
+import { Select } from "./select"
+
+const meta = {
+  component: Select,
+  parameters: {
```

Comment:
Please, remove ` layout: "centered"`  configuration from here and move it to general configuration of storybook

Reply:
fixed